### PR TITLE
docs: deploy `helm` with name `kubernetes-dashboard`

### PR DESCRIPTION
**Pull request by JounQin**

### CODE REVIEW

File: aio/deploy/helm-chart/kubernetes-dashboard/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To install the [Chart](https://helm.sh/docs/intro/using_helm/#three-big-concepts
 
 ```console
 helm repo add kubernetes-dashboard https://kubernetes.github.io/dashboard/
-helm install kubernetes-dashboard kubernetes-dashboard/kubernetes-dashboard
+helm install my-release kubernetes-dashboard/kubernetes-dashboard
```
Comment (desaintmartin) [inline aio/deploy/helm-chart/kubernetes-dashboard/README.md:25]: We currently have a limitation: some secret names are actually hardcoded in dashboard code: https://github.com/kubernetes/dashboard/blob/fe4b5fc1f4851136e868f52dbd78299b0529da79/src/app/backend/client/api/types.go#L33

Thus, to do this, we either have to:
- un-hardcode those names
- hardcode the secret names in the chart

Comment (JounQin) [reply aio/deploy/helm-chart/kubernetes-dashboard/README.md:25]: https://github.com/kubernetes/dashboard/blob/d0cfa46b287ab23bb0174356437b2818f2df37c9/aio/deploy/helm-chart/kubernetes-dashboard/README.md?plain=1#L21

The title is said `To install the Chart with the Release name my-release:`, maybe we should change `my-release` to `kubernetes-dashboard` instead for now?

Comment (desaintmartin) [reply aio/deploy/helm-chart/kubernetes-dashboard/README.md:25]: Indeed!
Please bump patch version of chart.yaml since chart is immutable.

Comment (JounQin) [reply aio/deploy/helm-chart/kubernetes-dashboard/README.md]: Done!

**Comment by desaintmartin** [comment]
Thanks!
/lgtm

**Pull request merged**